src/directives/share-button/share-button.directive.ts

```diff
@@ -8,80 +8,55 @@ import {
     EventEmitter
 } from '@angular/core';
 
-import { ShareButtonsService } from '../../service/share-buttons.service';
-import { ShareButton, ShareArgs, ShareProvider, Helper } from '../../helpers';
-
+import { ShareButtonsService } from '../../services/share-buttons.service';
+import { ShareArgs } from '../../helpers';
 
 @Directive({
     selector: '[shareButton]'
 })
 export class ShareButtonDirective implements OnChanges {
 
-    private _provider: ShareProvider;
-
-    /** Share Args */
-    @Input() url: string;
-    @Input() title: string;
-    @Input() description: string;
-    @Input() image: string;
-    @Input() tags: string;
-
     /** Button type e.g. fb, twitter, reddit...etc */
-    @Input() button: ShareButton;
-    /** Show count, disabled by default */
-    @Input() count: boolean = false;
+    @Input() shareButton: string;
+    /** Share Args */
+    @Input() sbUrl: string;
+    @Input() sbTitle: string;
+    @Input() sbDesc: string;
```

Comment:
honestly, i don't think `sbDescription` is too long... nor `sbImage` . there are clearer